client/signup/main.jsx

```diff
@@ -762,6 +768,7 @@ class Signup extends Component {
 								flowName: this.props.flowName,
 								stepName: this.props.stepName,
 							} }
+							pageTitle={ this.getPageTitle() }
```

Comment:
```suggestion
							pageTitle={ this.props.pageTitle }
```

Also, may need something to hide this if the nav buttons are present

<img width="672" alt="Markup 2022-09-11 at 19 53 31" src="https://user-images.githubusercontent.com/33258733/189542175-b29619f3-97af-4980-bedc-e2bea49ea0ad.png">

Reply:
Good catch! thanks!

Reply:
Do we want to show the page title in every flow, if no back buttons are present? 
If so I think we can use `pageTitle={ this.props.pageTitle }`, otherwise it'd be good to maintain the check that @renancarvalho added :)

Reply:
I think it is the other way around, we just show the page title for tailored flow( LiB and NL ), the 'normal' is the back button which has two 'versions' one 'back to sites' and the other one just 'back'. ( just kept the original behaviour )

@alshakero how do you think it should be?

Reply:
We shouldn't touch any other flows. Flows are A/B tested to the punctuation levels, we can't just change stuff. Let's keep the check.